bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/core/listeners/LimitsListener.java

```diff
@@ -36,6 +41,17 @@ public void propertyChange(PropertyChangeEvent evt) {
 		MarkerManager mm = v.getSchedule().getMarkerManager();
 		mm.clearMarkers(this, v);
 
+		final TimeZone zone = v.getSchedule().getSite().timezone();
+		final DateTimeFormatter dateFormat = DateTimeFormatter.ofPattern("HH:mm") .withZone(zone.toZoneId());
+
+		final Site site = v.getSchedule().getSite();
+
+		SortedSet<Block> blocks = v.getSchedule().getBlocks();
+		Interval night = null;
```

Comment:
It's a tiny wee detail, but `blocks` could be marked `final` here as well.  Also, we could use a Java `Optional` or else our own `edu.gemini.shared.util.immutable.Option` to avoid a nullable variable.